tests/unit/executor/test_organize.py

```diff
@@ -0,0 +1,227 @@
+                (["dir1", "dir2"], "dir"),
+                (["foo"], os.path.join("dir", "dir1")),
+                (["bar"], os.path.join("dir", "dir2")),
+            ],
+            expected_message=None,
+            expected_overwrite=None,
+        ),
+        # combined_*_with_file
+        dict(
+            setup_dirs=["dir1", "dir2"],
+            setup_files=[
+                os.path.join("dir1", "foo"),
+                os.path.join("dir1", "bar"),
+                os.path.join("dir2", "bar"),
+            ],
+            organize_map={"dir*": "dir/", "dir1/bar": "."},
+            expected=[
+                (["bar", "dir"], ""),
+                (["dir1", "dir2"], "dir"),
+                (["foo"], os.path.join("dir", "dir1")),
+                (["bar"], os.path.join("dir", "dir2")),
+            ],
+            expected_message=None,
+            expected_overwrite=None,
+        ),
+        # *_into_dir
+        dict(
+            setup_dirs=["dir"],
+            setup_files=[
+                os.path.join("dir", "foo"),
+                os.path.join("dir", "bar"),
+            ],
+            organize_map={"dir/f*": "nested/dir/"},
+            expected=[
+                (["dir", "nested"], ""),
+                (["bar"], "dir"),
+                (["dir"], "nested"),
+                (["foo"], os.path.join("nested", "dir")),
+            ],
+            expected_message=None,
+            expected_overwrite=None,
+        ),
+    ],
+)
+def test_organize(new_dir, data):
+    _organize_and_assert(
+        tmp_path=new_dir,
+        setup_dirs=data["setup_dirs"],
+        setup_files=data["setup_files"],
+        organize_map=data["organize_map"],
+        expected=data["expected"],
+        expected_message=data["expected_message"],
+        expected_overwrite=data["expected_overwrite"],
+        overwrite=False,
+    )
+
+    # Verify that it can be organized again by overwriting
+    _organize_and_assert(
+        tmp_path=new_dir,
+        setup_dirs=data["setup_dirs"],
+        setup_files=data["setup_files"],
+        organize_map=data["organize_map"],
+        expected=data["expected"],
+        expected_message=data["expected_message"],
+        expected_overwrite=data["expected_overwrite"],
+        overwrite=True,
+    )
+
+
+def _organize_and_assert(
+    *,
+    tmp_path: Path,
+    setup_dirs,
+    setup_files,
+    organize_map,
+    expected: List[Any],
+    expected_message,
+    expected_overwrite,
+    overwrite,
+):
+    base_dir = Path(tmp_path / "install")
+    base_dir.mkdir(parents=True, exist_ok=True)
+
+    for directory in setup_dirs:
+        (base_dir / directory).mkdir(exist_ok=True)
+
+    for file_entry in setup_files:
+        (base_dir / file_entry).touch()
+
+    if overwrite and expected_overwrite is not None:
+        expected = expected_overwrite
+
+    if isinstance(expected, type) and issubclass(expected, Exception):
+        with pytest.raises(expected) as error:  # type: ignore
```

Comment:
```suggestion
        with pytest.raises(expected) as exc_info:  # type: ignore
```

Reply:
I'm using `raised` in all other tests so I'll update this one to be consistent.